src/densities.jl

```diff
@@ -97,6 +97,37 @@ is not collinear the spin density is `nothing`.
     G_to_r(basis, ρ)
 end
 
+# The variation in density corresponds to a variation in the orbitals.
```

Comment:
Variation in density corresponding to a variation in the orbitals and occupations